core/sys/windows/kernel32.odin

```diff
@@ -1210,6 +1213,15 @@ SYSTEM_LOGICAL_PROCESSOR_INFORMATION :: struct {
 	DummyUnion: DUMMYUNIONNAME_u,
 }
 
+SYSTEM_POWER_STATUS :: struct {
+	ACLineStatus:        BYTE,
+	BatteryFlag:         BYTE,
+	BatteryLifePercent:  BYTE,
+	SystemStatusFlag:    BYTE,
+	BatteryLifeTime:     DWORD,
+	BatteryFullLifeTime: DWORD,
```

Comment:
`The number of seconds of battery life remaining, or –1 if remaining seconds are unknown or if the device is connected to AC power.`

Strange that they're they're using an unsigned type (`DWORD`), but insist that `-1` is used for `unknown`. Does that imply it's actually signed, or is it only `max(DWORD)` that has a special meaning?

Again not your fault. I just wish that the documentation was clearer.

Reply:
Thanks for the suggestions, it should be quite a bit nicer to use now. Btw I didnt add an enum/bitset for the system status flag, because I dont really think it really helps since there are only two possible values 0/1.